test/e2e/nvidia-container-toolkit_test.go

```diff
@@ -18,13 +18,25 @@ package e2e
 
 import (
 	"context"
+	"fmt"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 )
 
 // Integration tests for Docker runtime
-var _ = Describe("docker", func() {
+var _ = Describe("docker", Ordered, func() {
+	// Install the NVIDIA Container Toolkit
+	BeforeAll(func(ctx context.Context) {
+		if sshKey != "" {
```

Comment:
As I mentioned, we should probably allow the isntallation logic to be run locally too.